docs/data-sources/vpcs.md

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Virtual Private Cloud (VPC)"
+---
+
+# huaweicloud_vpc
+
+Use this data source to get a list of VPC.
+
+## Example Usage
+
+An example filter by name and tag
+
+```hcl
+variable "vpc_name" {}
+
+data "huaweicloud_vpcs" "vpc" {
+  name = var.vpc_name
+  tags {
+    foo = "bar,value"
+  }
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available VPCs in the current region.
+ All VPCs that meet the filter criteria will be exported as attributes. This data source will fail if none are found.
```

Comment:
`this data source will fail if none are found`
should return empty instead of raising error for listing data source.